test/integration/check_navbar_links.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Checks that the links in the navbar on the homepage have the right URLs to redirect to
+ *
+ * Test cases: https://github.com/LLK/scratch-www/wiki/Most-Important-Workflows#Create_should_take_you_to_the_editor
+ */
+
+var tap=require('tap'),
+    seleniumWebdriver = require('selenium-webdriver');
+
+/*
+ * Remove question comments after resolving them
+ */
+//how to generalize for other browsers... how will this work in saucelabs? Do I need to iterate 
+//through the drivers for each browser here?
+//chrome driver
```

Comment:
Eventually maybe we want to set the browser through an environmental variable and then iterate through running all the tests by switching the variable?  For now I think just one browser is ok.